.github/ISSUE_TEMPLATE/bug-report.yaml

```
@@ -0,0 +1,88 @@
+name: Bug report
+description: Report incorrect behavior in the Modin library
+title: 'BUG: '
+labels: ['bug 🦗', 'Triage 🩹']
+
+body:
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Modin version checks
+      options:
+        - label: >
+            I have checked that this issue has not already been reported.
+          required: true
+        - label: >
+            I have confirmed this bug exists on the latest released version of Modin.
+          required: true
+        - label: >
+            I have confirmed this bug exists on the main branch of Modin. (In order to do this you
+            can follow [this guide](https://modin.readthedocs.io/en/stable/getting_started/installation.html#installing-from-the-github-master-branch).)
+  - type: textarea
+    id: example
+    attributes:
+      label: Reproducible Example
+      description: >
+        Please follow [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) on how to
+        provide a minimal, copy-pastable example.
+      placeholder: >
+        import modin.pandas as pd
+
+        df = pd.DataFrame(range(5))
+
+        ...
+      render: python
+    validations:
+      required: true
+  - type: textarea
+    id: problem
+    attributes:
+      label: Issue Description
+      description: >
+        Please provide a description of the issue shown in the reproducible example.
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: >
+        Please describe or show a code example of the expected behavior.
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Error Logs
+      description: >
+        Please paste the output of any relevant error logs.
+      value: >
+        <details>
+
+
+        ```python-traceback
+
+
+        Replace this line with the error backtrace (if applicable).
+
+
+        ```
+
+
+        </details>
+  - type: textarea
+    id: version
+    attributes:
+      label: Installed Versions
+      description: >
+        Please paste the output of ``pd.show_versions()``
+      value: >
+        <details>
+
+
+        Replace this line with the output of pd.show_versions()
+
+
+        </details>
+    validations:
+      required: true
```

```suggestion
      required: true

```
There is no end empty line probably.

---

Does this matter? It seems to be rendering just fine?

---

In the removed files we have had an empty line, so just wanted to be aligned and not to have such red marks :)